src/com/itschool/session4/homework/challenge03/LogicalOperators.java

```diff
@@ -0,0 +1,25 @@
+package com.itschool.session4.homework.challenge03;
+
+import java.util.Scanner;
+public class LogicalOperators {
+
+    public static void main(String[] args) {
+        //creaza un nou obiect  Scanner
+        Scanner obiecScaner = new Scanner(System.in);
+
+
+        // Prompt pentru introducere numar
+        System.out.println("Introduceti primul numar");
+        int numar1 = obiecScaner.nextInt();
+
+        boolean isPositive = !(numar1<0);
+
+        if(isPositive){
+            System.out.println("Numarul este pozitiv");
+        } else{
+            System.out.println("Numarul este negativ");
+        }
+
+        obiecScaner.close();
```

Comment:
Can you explain what is the purpose of this/ what is does and why did you use it here? I see you don't have this in other challenges.

Reply:
Indeed, but here you are not using any external resource nor network streams, but the builtin standard JVM input, therefore there is no chance at all to get stuff broken. Moreover, the program exits once this execution here is finished so all the stuff gets terminated (including eventually opened resources).
To summarize, it is not wrong to have this line in this situation, but also it does not make much of a difference, being kinda useless.